service/controller/resource/tccpsecuritygroups/resource.go

```diff
@@ -67,6 +67,12 @@ func (r *Resource) addInfoToCtx(ctx context.Context, cr infrastructurev1alpha2.A
 						aws.String(key.SecurityGroupName(&cr, "master")),
 					},
 				},
+				{
+					Name: aws.String(key.TagStack),
+					Values: []*string{
+						aws.String(key.StackTCCP),
+					},
+				},
```

Comment:
i found a bug where the `cc.Status.TenantCluster.TCCP.SecurityGroups` and `cc.Status.TenantCluster.TCCP.Subnets` could be filled with resources from TCNP which caused some nasty issues